instrumentation/netty/netty-4.1/src/test/resources/logback.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+  <appender name="console" class="ch.qos.logback.core.ConsoleAppender">
+    <layout class="ch.qos.logback.classic.PatternLayout">
+      <Pattern>
+        %d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n
+      </Pattern>
+    </layout>
+  </appender>
+
+  <root level="WARN">
+    <appender-ref ref="console"/>
+  </root>
+
+  <logger name="io.opentelemetry.auto" level="debug"/>
```

Comment:
And this logback.xml like many of ours is wrong :-D /cc @trask

Reply:
Btw, it was not wrong! `InMemoryExporter` still lives in `io.opentelemetry.auto.test` package!

Reply:
well, it's not _totally_ wrong then 😂

fixed in #1194